### PR TITLE
Changing add-on with a single author to be MemChaser

### PR DESCRIPTION
**Pull request by stephendonner**
Fixes Issue#769, I hope.

Should have a run on the staging Jenkins soon to report back on.

### CODE REVIEW

**Comment by davehunt** [comment]
r+

**Comment by stephendonner** [comment]
Thx; I ran this against both dev + stage.

File: tests/desktop/test_details_page.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def test_other_addons_label_when_there_are_multiple_authors(self, mozwebqa):
 
     @pytest.mark.nondestructive
     def test_other_addons_label_when_there_is_only_one_author(self, mozwebqa):
-        addon_with_one_authors = 'F1 by Mozilla Labs'
+        addon_with_one_authors = 'MemChaser'
```
Comment (bobsilverberg) [inline tests/desktop/test_details_page.py]: @stephendonner I am surprised you didn't also take the opportunity to change the variable name to `addon_with_one_author`. ;)

Comment (davehunt) [inline tests/desktop/test_details_page.py]: that'll be in a separate PR :wink:

**Comment by stephendonner** [comment]
Indeed it is: https://github.com/mozilla/Addon-Tests/pull/772 ;-)
